client/mm/libxc/bntypes/types.go

```diff
@@ -34,9 +57,9 @@ type NetworkInfo struct {
 	// ResetAddressStatus      bool    `json:"resetAddressStatus"`
 	// SpecialTips             string  `json:"specialTips"`
 	// UnLockConfirm           int     `json:"unLockConfirm"`
-	WithdrawEnable bool    `json:"withdrawEnable"`
-	WithdrawFee    float64 `json:"withdrawFee,string"`
-	// WithdrawIntegerMultiple float64 `json:"withdrawIntegerMultiple,string"`
+	WithdrawEnable          bool    `json:"withdrawEnable"`
+	WithdrawFee             float64 `json:"withdrawFee,string"`
+	WithdrawIntegerMultiple float64 `json:"withdrawIntegerMultiple,string"`
 	// WithdrawMax             float64 `json:"withdrawMax,string"`
 	WithdrawMin float64 `json:"withdrawMin,string"`
 	// SameAddress             bool    `json:"sameAddress"`
```

Comment:
Maybe remove commented.

Reply:
I think it's ok.. good to know what other fields there are without having to check the API.